source/ada/lsp-ada_handlers.adb

```diff
@@ -2494,11 +2494,33 @@ package body LSP.Ada_Handlers is
       Sloc : constant Langkit_Support.Slocs.Source_Location :=
         Document.Get_Source_Location (Value.position);
 
-      Name_Node       : Libadalang.Analysis.Name;
-      Designator      : Libadalang.Analysis.Ada_Node;
-      Active_Position : LSP.Types.LSP_Number;
+      Name_Node        : Libadalang.Analysis.Name;
+      Designator       : Libadalang.Analysis.Ada_Node;
+      Active_Position  : LSP.Types.LSP_Number;
+      Active_Signature : LSP.Types.LSP_Number := 0;
 
       procedure Add_Signature (Decl_Node : Libadalang.Analysis.Basic_Decl);
+      --  Add the signature of Decl_Node if it matches the current context.
+
+      procedure Filter_Signature
+        (Signature_Info : SignatureInformation;
+         Designator     : Libadalang.Analysis.Ada_Node;
+         Position       : LSP.Types.LSP_Number;
+         Is_Active      : Boolean);
+      --  Filter the signatures returned by the client using the current
+      --  current Position and designator.
```

Comment:
should be removed

Reply:
Remove the comment or replace "filter" by "remove"